public/app/percona/add-instance/components/AzureDiscovery/Discovery.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, useEffect, useState } from 'react';
+import DiscoveryService from './Discovery.service';
+import Credentials from './components/Credentials/Credentials';
+import Instances from './components/Instances/Instances';
+import { getStyles } from './Discovery.styles';
+import { DiscoverySearchPanelProps } from './Discovery.types';
+import { AzureCredentialsForm } from './components/Credentials/Credentials.types';
+
+const Discovery: FC<DiscoverySearchPanelProps> = ({ selectInstance }) => {
+  const styles = getStyles();
+
+  const [instances, setInstances] = useState([] as any);
```

Comment:
There's a type for `Instace` just used that instead of `any`

Reply:
Done